### DISCUSSION TITLE
How to repeat play_sound or How to cancel a play

### DISCUSSION THREAD

**Discussion by mjafartp** [halildurmus/win32#358] [Q&A] [answered]
Is there an option to repeat sound play and cancel a playing sound?

**Comment by timsneath** [comment, marked as answer]
The Win32 documentation describes how to loop a sound (so it repeats) and to stop a playing sound here: 
https://docs.microsoft.com/en-us/windows/win32/multimedia/using-playsound-to-loop-sounds

The two code samples should essentially work unchanged in Dart compared to the C++ examples.

**Comment by mjafartp** [comment]
It's worked.
